conf/bookkeeper.conf

```diff
@@ -158,6 +158,10 @@ numReadWorkerThreads=8
 # (i.e. recovery reads and adds, and fencing).
 numHighPriorityWorkerThreads=8
 
+# This is the number of threads used by Netty to handle TCP connections.
+# Default is 2 *  Runtime.getRuntime().availableProcessors()
+serverNumIOThreads=
```

Comment:
The parameter name (`serverNumIOThreads`) is different from that of broker (`numIOThreads`). Does it make sense to keep it aligned?

Reply:
They are different parameters